lisa/transformers/kernel_source_installer.py

```diff
@@ -330,6 +330,11 @@ def get_source_code(self) -> PurePath:
             self._log.info(f"checkout code from: '{runbook.ref}'")
             git.checkout(ref=runbook.ref, cwd=code_path)
 
+        result = self._node.execute(
+            "git log --pretty=format:%h | head -1", cwd=code_path, shell=True
```

Comment:
please use git tool, if no such method, please add one, thanks. how do you plan to pass it into db?

Reply:
Added 'list_commit_ids()' to git tool. In pipeline this info is collected and pushed to logs db as tag.